warehouse/query-core/src/main/java/datawave/query/iterator/logic/IndexIterator.java

```diff
@@ -137,8 +137,7 @@ private IndexIterator(Text field, Text value, SortedKeyValueIterator<Key,Value>
         
         this.source = source;
```

Comment:
Consider making the `source` field `final`

Reply:
done